ui/store/actions.js

```diff
@@ -2216,6 +2216,19 @@ export function setAdvancedGasFee(val) {
   };
 }
 
+export function setTheme(val) {
+  return (dispatch) => {
+    dispatch(showLoadingIndication());
+    log.debug(`background.setMetamaskThemes`);
+    background.setTheme(val, (err) => {
```

Comment:
Nit: Both `background` and `displayWarning` should be considered deprecated. Please use `promisifiedBackground` instead of `background`, and find some other way to handle errors. e.g. if no error is expected, let it bubble up so that Sentry gets the report.